resources/remote-control.txt

```diff
@@ -1,21 +1,38 @@
 Remote control protocol.
 
 Supported commands:
- f - Get frequency [Hz]
```

Comment:
As I have said in the previous review, I do want to see changes in a commit that are not related to the topic in question. In this case the commit message says "Implement the M ? command" and so the commit should only contain that.

If you wish to reformat the text file, fine with me, but do it in a separate commit. I'm going to accept it this last time because it is a small change, but please remember it for the next time.